src/Util/Sortable.js

```diff
@@ -33,7 +35,7 @@ class Sortable {
       sortFunction = Sortable["sortFunction" + capitalize(sortFunction)];
     }
 
-    return this.items.filter(() => true).sort(sortFunction);
+    return this.items.slice().sort(sortFunction);
```

Comment:
Ah is `slice` better here? I think I do the filter thing in a few places

Reply:
`slice` should inline to skip iteration entirely, particularly for fast element arrays

Reply:
I haven't benchmarked this myself, but this post leads me to think the `slice()` is probably the right call for a pure clone.  Will update the PR to use that everywhere.

Reply:
Fixed.